readme.md

```diff
@@ -130,6 +130,11 @@ Some hosting providers filter this out using mod_security. For one user, they ne
 
 ## Upgrade Notice ##
 
+### 3.6.0 ###
```

Comment:
Having this is in the README is overly noisey. Changelog and Upgrade Notice are only going to grow and grow and grow.  Could we shift them out to UPGRADE and CHANGELOG respectively?

Reply:
The `README.md` is generated from the `readme.txt`, so that's why it is part of the readme. I would not change that in the first place, because then we have to write a tool or do a lot by hand.

Reply:
The various sections of the file are used by WordPress Core in the readme.txt file, so they need to be there. I suppose if we wanted to, we could put the UPGRADE, CHANGELOG, etc into separate files, and add a simple script that cats them together for the readme.txt but not for the readme.md. But...back to tooling.

Reply:
Would suggest, being as this is a unrelated to the PR at hand, we consider this separately from this revision.